### ISSUE TITLE
How to open the 25 files in ./checkpoints folder?

### ISSUE THREAD

**Issue by kiwi6185** [mysbupt/CrossCBR#2]
I'm trying to reproducing the paper running the code but wondering what software is used to open the 25 files which is the result.

**Comment by mysbupt** [comment]
The files in ./checkpoints are the dumped files of your trained model. These files are generated by this line of code: https://github.com/mysbupt/CrossCBR/blob/bbe80dfb1bbe878d72389185edefc655c8b5db45/train.py#L206, which just calls the standard pytorch function, i.e., torch.save(). To load a saved model from file, just call torch.load() will do. For more information about pytorch on model save and load, you can refer here https://pytorch.org/tutorials/beginner/saving_loading_models.html. 

In summary, to load a trained model from files in the ./checkpoint folder, just write a script using the torch.load() function. Hope this explanation is clear. Thanks for your interest in our work.

**Comment by kiwi6185** [comment]
Thank you for ur help. :)
